pymc/distributions/continuous.py

```diff
@@ -2771,15 +2771,18 @@ def dist(cls, mu=0.0, sigma=None, nu=None, sd=None, *args, **kwargs):
         sigma = at.as_tensor_variable(floatX(sigma))
         nu = at.as_tensor_variable(floatX(nu))
 
-        # sd = sigma
-        # mean = mu + nu
-        # variance = (sigma ** 2) + (nu ** 2)
-
         assert_negative_support(sigma, "sigma", "ExGaussian")
         assert_negative_support(nu, "nu", "ExGaussian")
 
         return super().dist([mu, sigma, nu], *args, **kwargs)
 
+    def get_moment(rv, size, mu, sigma, nu):
+        moment, _ = at.broadcast_arrays(sigma, nu)
+        moment = mu + nu
```

Comment:
This is probably what you want. Otherwise your first line is not doing anything as you overwriting moment after

```suggestion
        mu, nu, _ = at.broadcast_arrays(mu, nu, sigma)
        moment = mu + nu
```

You should also change one of the tests, making nu a vector and mu, sigma scalars and size is `None` to test the case where that first line is really needed

Reply:
Thanks for the suggestion, I have corrected `at.broadcast_arrays()` and added one test case for the same. But I am not sure about the test case if it is proper or not. Can you please check that?

Reply:
The test does not seem to cover the case(s) where `at.broadcast_arrays` would be necessary. You can check this by commenting out that line in `get_moment` and verifying that all tests still pass.

Reply:
Finally, I tried to make a test case that cover the `at.broadcast_arrays()` line, please look into that!